config/routes.rb

```diff
@@ -1,6 +1,40 @@
 Rails.application.routes.draw do
+  devise_for :users,controllers: {
+                 sessions: 'users/sessions',
+                 registrations: 'users/registrations'
+             }
+  get 'comments/new'
+  get 'comments/create'
+  get 'likes/create'
+  
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'users#index'
+  resources :users, only: [:index, :show ] do 
+    resources :posts, only: [:index, :show, :new, :create, :destroy] do
+    resources :comments, only: [ :create, :destroy]
+    resources :likes, only: [:create ]
+  end 
+end
+# namespace :api, defaults: { format: :json } do
+#   namespace :v1 do
+#   resources :users, only: [:index] do 
+#     resources :posts, only: [:index] do
+#       resources :comments, only: [:index] 
+#      end
+#     end
+#   end
+#   end
+
+namespace :api do 
+  namespace :v1 do 
+    resources :users, only: %i[index] do 
+      resources :posts, only: %i[index show] do 
+        resources :comments, only: %i[index new create ] 
+      end 
+    end 
+  end 
+end 
```

Comment:
- [ ] Hello and nice job on adding the API, however the API folder that you have created is not rendering any details like users or posts or comments all I do see is a message that I'm logged in and that is a good start, but everything else should be as this message example: { "users": "id":1, name .....} and the same for the comment.


![Capture d’écran 2022-08-27 163152](https://user-images.githubusercontent.com/61562626/187037088-c7963662-4a26-4531-972d-c45b6a52ba38.png)

Reply:
Please check the documentation in the pull request on how to use my API .:grr